lib/topic.ts

```diff
@@ -73,16 +73,16 @@ export const deleteKafkaTopic = async function (page: Page, name: string) {
 export const navigateToMessages = async function (page: Page, kafkaName: string, topicName: string) {
   await navigateToKafkaList(page);
   await navigateToKafkaTopicsList(page, kafkaName);
-  await expect(await page.locator('a', { hasText: topicName })).toHaveCount(1);
+  await expect(page.locator('a', { hasText: topicName })).toHaveCount(1);
   await page.locator('a', { hasText: topicName }).click();
-  await expect(await page.locator('button', { hasText: 'Messages' })).toHaveCount(1);
+  await expect(page.locator('button', { hasText: 'Messages' })).toHaveCount(1);
   await page.locator('button', { hasText: 'Messages' }).click();
 };
 
 export const navigateToProperties = async function (page: Page, kafkaName: string, topicName: string) {
   await navigateToKafkaList(page);
   await navigateToKafkaTopicsList(page, kafkaName);
-  await expect(await page.locator('a', { hasText: `${topicName}` })).toHaveCount(1);
+  await expect(page.locator('a', { hasText: `${topicName}` })).toHaveCount(1, { timeout: 6000 });
```

Comment:
Is it really waiting 30 seconds? Why does it fail then with 30 seconds and not with 6?

Reply:
Well, it should, see https://github.com/redhat-developer/consoledot-e2e/blob/main/playwright.config.ts#L23 ... maybe we interpret it in the wrong way?

Reply:
that is https://playwright.dev/docs/test-timeouts#expect-timeout Assertion timeout is unrelated to the test timeout. There are several different kinds of Timeout, this is acting on the lines of code that were repeatedly failing

Reply:
But you are setting timeout for expect to 6s but our default value is 30s. This is not related to test timeout that is set to 20 minutes.

Reply:
Expect timeout
Web-first assertions like expect(locator).toHaveText() have a separate timeout, 5 seconds by default. Assertion timeout is unrelated to the test timeout.

Reply:
https://github.com/redhat-developer/consoledot-e2e/blob/main/playwright.config.ts#L18

```
export default defineConfig({
  expect: {
    timeout: 10 * 1000,
  },
});
```

I don't think there is a difference between doc and our usage

Reply:
Can I resolve this conversation?

Reply:
No, because I still didn't get a proper explanation of why this is needed.

Reply:
do you understand what I am trying to say by all of the lines above? Is there some bug in pw that did not respect the configuration from the config file? Or why does changing the timeout form 30s to 6s solve it? Or it is somehow related to removing await - if so, why? The expected timeout should work fine anyway.

Reply:
Ok, so do you suggest I should only remove the unnecessary await and don't add the timeout?

Reply:
You can try it, all I am saying is, that your change basically doesn't change how the code should behave so it will probably don't fix the real problem. Unless we don't fully understand how pw should work, this might be true as well :) But I didn't get any proper explanation why your changes are working and the current code not